frontend/src/components/ApartmentCard/ApartmentCard.tsx

```diff
@@ -1,63 +1,114 @@
 import React, { ReactElement } from 'react';
-import ApartmentImg from '../../assets/apartment-sample.png';
-import { Card, CardContent, CardMedia, Grid, makeStyles, Typography } from '@material-ui/core';
+import ApartmentImg from '../../assets/apartment-placeholder.png';
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Grid,
+  makeStyles,
+  Typography,
+  useMediaQuery,
+} from '@material-ui/core';
 import { Apartment } from '../../../../common/types/db-types';
+import HeartRating from '../utils/HeartRating';
 
 type Props = {
   buildingData: Apartment;
   numReviews: number;
   company?: string;
 };
-
 const useStyles = makeStyles({
   img: {
-    height: 200,
+    borderRadius: '12%',
+    height: '205px',
     width: '100%',
+    padding: '17px',
+  },
+  msgImg: {
```

Comment:
Don't forget to delete`msgImg` and `msg.png` since they're not used. Otherwise, great job!!!! Wait for Jiayuan to merge #168 first before merging this PR.

Reply:
Doing rn, thanks! I will wait for Jiayuan.